modules/admin_manual/pages/configuration/server/excluded_blacklisted_files.adoc

```diff
@@ -3,68 +3,71 @@
 
 == Definitions of terms
 
-*Blacklisted*::
-  Files that may harm the ownCloud environment like a foreign
-  `.htaccess` file. Blacklisting prevents anyone from uploading
-  blacklisted files to the ownCloud server.
-*Excluded*::
-  Existing directories on your ownCloud server, including external
-  storage mounts, that are excluded from being processed by ownCloud. In
-  effect they are invisible to ownCloud.
-
-Both types are defined in `config.php`. Blacklisted files and excluded
-directories are not scanned by ownCloud, not viewed, not synced, and
-cannot be created, renamed, deleted, or accessed via direct path input
-from a file explorer. Even when a filepath is entered manually via a
-file explorer, the path cannot be accessed.
-
-For example configurations please see
-`owncloud/config/config.sample.php`.
+Blacklisted::
+Files that may harm the ownCloud environment like a foreign `.htaccess` file. 
```

Comment:
What do you mean by "foreign .htaccess file"?

Reply:
A .htaccess file not being part of ownCloud